transactions.md

```diff
@@ -6,9 +6,60 @@ One transaction event occurs at authorisation time. The other transaction event
+# Transaction Object - Select Transactions
+
+| **Field name** | **Description** | **Type** |
+| --- | --- | --- |
+| accountId | The unique identifier of the user account at Fidel API | string |
+| amount | The amount of the transaction in the currency it was charged in | number |
+| approvalCode | Unique code that is sent along with the authorization of the transaction by Amex. It mirrors the identifiers.amexApprovalCode property. | string (or null) |
+| auth | Indicates whether Fidel API received an authorization event for this transaction | boolean |
+| authCode | Unique code that is sent along with the authorization of the transaction by Visa and Mastercard. It is sent by the issuer/the PCN on the issuer's behalf when approving a transaction. It mirrors the identifiers.mastercardAuthCode or identifiers.visaAuthCode properties, depending on the issuing card for the transaction. | string (or null) |
+| brand.id | The identifier of the brand at Fidel API | string |
+| brand.logoURL | The URL of the logo for the brand | string (or null) |
+| brand.name | The name of the merchant where the purchase was made | string |
+| brand.metadata | Customized information you added to the brand | (Can be null) |
+| card.firstNumbers | The first numbers of the card, used for helping users identify their cards | string |
+| card.id | The tokenized card identifier created by Fidel API | string |
+| card.lastNumbers | The last numbers of the card, used for helping users identify their cards | string |
+| card.metadata | Customized information you added to the card | (Can be null) |
+| card.scheme | The payment network (Visa, Mastercard or Amex) | string |
+| cardPresent | Indicates whether the transaction was in store (as opposed to online). Only available in Visa transactions, and only if the transaction data received from Visa contains this information. In test transactions, this field is always null. | boolean (or null) |
+| cleared | Indicates whether Fidel API received a clearing event for this transaction | boolean |
+| created | The time when the transaction object was created in the Fidel API database | string, in ISO format |
+| currency | The currency of the purchase | string, in ISO 4217 format |
+| datetime | The time of the transaction (local time of the merchant) | string, in ISO format |
+| id | The unique identifier of the transaction | string |
+| identifiers.amexApprovalCode | Unique code that is sent along with the authorization of the transaction by Amex. It mirrors the approvalCode property. | string (or null) |
+| identifiers.mastercardAuthCode | Unique code that is sent along with the authorization of the transaction by Mastercard. It mirrors the authCode property. | string (or null) |
+| identifiers.mastercardRefNumber | Unique identifier of the transaction provided by Mastercard. This is used by Fidel API to try to match a given clearing transaction with the corresponding authorisation transaction. | string (or null) |
```

Comment:
"authorization" (with a z) is used in the whole table, so I would change it in this line too, even if the rest of the page uses British English.

Reply:
Updated 👍